src/main.cpp

```diff
@@ -241,7 +244,10 @@ class Konstructs: public nanogui::Screen {
         glfwGetFramebufferSize(mGLFWWindow, &width, &height);
 
         ostringstream os;
-        os << "Connected to " << hostname << " with user " << username;
+        if (debug_text_enabled) {
+            os << "Connected to " << hostname << " with user " << username;
+        }
+        glActiveTexture(GL_TEXTURE0);
```

Comment:
Note that I had to set the active texture here, with out this the screen went black again when I changed the displayed text length.